styles.css

```diff
@@ -80,34 +99,82 @@ Forms Page CSS Begins
     border-left-style: hidden;
     border-bottom-style: groove;
 }
-#stepOne, #stepTwo, #stepThree {
+
+#stepOne,
+#stepTwo,
+#stepThree {
     width: 100%;
     display: flex;
 }
-#stepTwo, #stepThree {
+
+#stepTwo,
+#stepThree {
     display: none;
 }
-.leftDiv, .rightDiv {
+
+.leftDiv,
+.rightDiv {
     float: left;
     width: 50%;
     margin: 20px;
     align-items: center;
 }
+
 .leftDiv {
     text-align: left;
 }
+
 .fieldset {
     line-height: 50px;
     width: 100%;
     border-radius: 15px;
 }
+
 textarea {
     width: 80%;
     height: 100px;
 }
+
 .leftDiv h3 {
     text-align: center;
 }
+
 /*******************************************
 Forms Page CSS Ends
 ******************************************/
+/* Grant Session */
+
+.jumbotron-horn {
+
+    color: #ffffff;
+}
+
+.btn {
+    background-color: #10374A;
+    ;
+    color: #ffffff;
+}
+
+.container-title {
+    margin: 40px;
+    font-weight: 600;
+    color: #10374A;
+}
+
+.btn:hover {
+    color: grey;
+}
+
+.bg-purple {
+    background-color: #14A2B8 !important;
+    color: #FFF;
+}
+
+.bg-red {
```

Comment:
Are we using red colours now?

Reply:
i used it for the card, when you merge and pull you will see it